### PR TITLE
Convert the project to use Xcode 8.3 , and move to Fastlane match.

### PR DESCRIPTION
**Pull request by orta**
Blog post incoming.

### CODE REVIEW

**Comment by orta** [comment]
post: https://github.com/artsy/artsy.github.io/pull/337

File: .bundle/config [hunk 1]
```diff
@@ -1,2 +1,2 @@
 ---
-BUNDLE_WITHOUT: deployment
+BUNDLE_WITH: "deployment"
```
Comment (jonallured) [inline .bundle/config]: I'm confused on this one - shouldn't this be the default behavior? Unless I'm missing something, `BUNDLE_WITH` doesn't appear on the list here:

http://bundler.io/v1.14/man/bundle-config.1.html

So I guess I think this config is being ignored? Probably safe to just remove the file altogether, no?

File: Supporting Files/Info.plist [hunk 2]
```diff
@@ -32,7 +32,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>2016.01.07</string>
+	<string>2017.04.04.15</string>
```
Comment (jonallured) [inline Supporting Files/Info.plist]: Hrm, is this meant to be a date string? Not sure what happened here.

Comment (orta) [inline Supporting Files/Info.plist]: Yeah, they are, probably got updated so we can deploy a beta more than once per hour

File: .bundle/config (hunk 1 above)
Comment (orta) [inline .bundle/config]: aye, IMO, yes

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by orta** [comment]
Alright - updated 👍

**Pull request merged**